packages/web3-contracts/abis/DODORouteProxy.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
Let's remove unused methods here to keep this file as thin as possible.

Reply:
done